### ISSUE TITLE
Add (soon to be) new OPNsense icon

### ISSUE THREAD

**Issue by EthanBezz** [homarr-labs/dashboard-icons#846]
### Contribution Guidelines

- [X] I have read and agree to the [Contribution Guidelines](https://github.com/walkxcode/dashboard-icons/blob/main/CONTRIBUTING.md)

### Icon Name

OPNsense

### Icon(s)

![opnsense-alt](https://github.com/user-attachments/assets/037268b6-5744-46c6-a8fe-197c02670231)
![opnsense-alt](https://github.com/user-attachments/assets/ed537b5f-4c41-4a08-99d3-a2b182b62345)

**Comment by EthanBezz** [comment]
While this new icon won't be added to OPNsense until the next major release (version 25.1 - expected end of January), it is already in use in a few places, including the [OPNsense forum](https://forum.opnsense.org/index.php)

For more info: https://www.reddit.com/r/opnsense/comments/1hbvz0n/ongoing_ci_rebranding_hitting_the_gui_in_opnsense/

**Comment by stuntguy3000** [comment]
@walkxcode Does this attached SVG File work for you? I can download it, but it does not have any colour :(

**Comment by EthanBezz** [comment]
@stuntguy3000 Hey I'm the OP. I think I've fixed it. How about now?

![opnsense](https://github.com/user-attachments/assets/15ff9a20-be05-4e5c-8214-bd1ee973c6b8)

**Comment by stuntguy3000** [comment]
@EthanBezz How does that file render in your browsers, if you open it directly?

**Comment by EthanBezz** [comment]
@stuntguy3000 Yeah I see your problem now. I have no colour either when viewed in Safari or Firefox. Once saved locally, it does have colour, even when viewed in the same browsers.

**Comment by stuntguy3000** [comment]
> @stuntguy3000 Yeah I see your problem now. I have no colour either when viewed in Safari or Firefox. Once saved locally, it does have colour, even when viewed in the same browsers.

Weird! See if you can fix it up, otherwise I can take a crack in the next day or two

**Comment by EthanBezz** [comment]
@stuntguy3000 Okay it's definitely fixed now. I edited my comment above with the fixed SVG so I could test it, so you can just grab it from there.

**Comment by stuntguy3000** [comment]
Looks good! Will PR in a few hours.
